frontend/src/components/Contributions/OpenCreationsAmount.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="open-creations-amount">
+    <b-table striped hover :items="items"></b-table>
+  </div>
+</template>
+<script>
+export default {
+  name: 'OpenCreationsAmount',
+  data() {
+    return {
+      items: [
+        { monat: 'Oktober', status: 'Max. erreicht', eingereicht: '50 h', stundenOffen: '0 h' },
+        { monat: 'November', status: 'einreichen', eingereicht: '20 h', stundenOffen: '30 h' },
```

Comment:
this functionality must be correctly implemented

Reply:
done